### DISCUSSION TITLE
Upgrade to the specified version

### DISCUSSION THREAD

**Discussion by airsoup** [fwupd/fwupd#5808] [General]
**Describe the question**
Can I choose the specified version instead of the latest version when updating?

**fwupd version information**

```shell
fwupdmgr --version
runtime   org.freedesktop.fwupd         1.8.10
runtime   com.hughsie.libjcat           0.1.11
compile   org.freedesktop.gusb          0.3.10
runtime   org.kernel                    5.19.0-41-generic
compile   com.hughsie.libjcat           0.1.11
runtime   org.freedesktop.gusb          0.4.0
compile   org.freedesktop.fwupd         1.8.10
```

**Comment by hughsie** [comment]
If you do `fwupdmgr get-releases` you can see all the options, and then `fwupdmgr install` with no arguments will let you choose:

	[hughsie@hughsie-work build (main *%)]$ fwupdmgr install
	0.	Cancel
	1.	2292ae5236790b47884e37cf162dcf23bfcd1c60 (Embedded Controller)
	2.	349bb341230b1a86e5effe7dfe4337e1590227bd (Intel Management Engine)
	3.	a45df35ac0e948ee180fe216a5f703f32dda163f (System Firmware)
	4.	362301da643102b9f38477387e2193e57abaa590 (UEFI dbx)
	5.	08e1798bf5d9cb56a0290b552cab6c1a371b5089 (WDC PC SN720 SDAQNTW-256G-1001)
	Choose device [0-5]: 3
	0.	Cancel
	1.	0.1.27
	2.	0.1.26
	3.	0.1.25
	4.	0.1.24
	5.	0.1.23
	6.	0.1.22
	7.	0.1.21
	Choose release [0-7]:

**Comment by airsoup** [comment]
Thank you for your reply. It's useful. Have a nice day.
